TuneUp/Properties/AssemblyInfo.cs

```diff
@@ -33,5 +33,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("2.5.0.0")]
```

Comment:
I think we should note on this repo somewhere - that his project is not semantically versioned at this time and should not be built on top of. I'm not sure how to convey this with a version number, but aligning it with Dynamo's version number conveys the opposite to me.

I guess it is rather useful to know which version of Dynamo this build of tuneup works with though.

Reply:
Yes, this is exactly what I am trying to log to, but other than assembly version, I do not have a good place to record such info, maybe branching it?